README.md

```diff
@@ -1,3 +1,13 @@
 # Replicated Helm Charts
 
 Helm charts maintained by the Replicated team
+
+## Releasing
+
+This repo uses [chart-releaser](https://github.com/helm/chart-releaser) to release all charts in the charts folder. Chart releaser will look for changes in the charts folder, and release charts that have changed on merge to main. The release is made available on github using the version that the charts `Chart.yaml` lists. The repository will also be automatically tagged for the release making it easy to identify where in the source history any given chart version was created from.
+
+While chart-releaser will release multiple charts at once, pull requests should only modify a single chart at a time. If any chart fails to release chart-releaser will stop processing charts. By only modifying a single chart with any given pull request, not only is version history clear for each chart but a failure to release one chart does not impact the release of an alternate chart.
+
+The Github action will list any reason for failure to release. A common mistake is not updating the chart.lock file to match the Chart.yaml file. If a lock file is included in a chart chart-release will verify the it matches the Chart.yaml version to avoid unexpected dependency versions. A failure to release only needs to be fixed in a new PR and merged again to trigger chart-releaser to try again.
```

Comment:
```suggestion
The Github action will list any reason for failure to release. A common mistake is not updating the Chart.lock file to match the Chart.yaml file. If a lock file is included in a chart, chart-release will verify that it matches the Chart.yaml version to avoid unexpected dependency versions. A failure to release only needs to be fixed in a new PR and merged again to trigger chart-releaser to try again.
```